DetectorDescription/Detectors/compact/FCChh_ECalDefinition.xml

```diff
@@ -9,3 +9,10 @@
     <passive_layers thickness="5*mm" material="Lead" />
   </calorimeter>
 </detector>
+
+  <readouts>
+    <readout name="BarECal_Readout">
+      <segmentation type="CartesianGridXY" grid_size_x="0.05*mm" grid_size_y="0.05*mm"/>
+      <id>system:5,layer:50,x:32:-16,y:-16</id>
```

Comment:
Why do you redefine the readout here? Also: The ID is a 64 bit number of which you define 50 to be taken for layer identification and 32 for segmentation, which doesn't work out. I suspect that this could have to do with the time out.